### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by KaustubhM1997**
## Landing page for e-commerce store

Live Preview: https://nervous-jepsen-33fba8.netlify.app/

Pages and Routes added:
Home page
Login (dummy page)
Products (dummy page)
Cart (dummy page)
Wishlist (dummy page)

Components:
Navbar
Footer

Assets:
Images

Please note: the pages are yet to be made responsive

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,13 +2,22 @@ import "./App.css";
 import logo from "./logo.png";
 import {Routes, Route, Link} from "react-router-dom";
 import { Home } from "../src/pages/Home/Home";
+import {ProductListing} from "../src/pages/Product-listing/product-listing"
+import { Wishlist } from "./pages/Wishlist/wishlist";
+import {Cart} from "./pages/Cart/cart"
+import {Login} from "./pages/Login/login"
```
Comment (srishti-maurya) [inline src/App.js]: you may use barrel exports for the pages and components

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,97 +1,128 @@
...
+            <p className="card-heading">Health and Personal Care</p>
 
-          <img className="img-card adjust" src="/assets/kids-nav.webp" alt="" />
-        </div>
+            <img
+              className="img-card adjust"
+              src="/assets/health-personal-care.jpg"
+              alt=""
+            />
+          </div>
```
Comment (srishti-maurya) [inline src/pages/Home/Home.jsx]: you may create this as a component, and reuse them

**Review by srishti-maurya — APPROVED** [review]
You may format your files for better indentation and readability.
Rest everything looks good!

File: src/App.js [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
-
-      <Route path= "/" element = {<Home/>}></Route>
-
-
+      <Route path="/" element={<Home />}></Route>
+      <Route path="/product-listing" element={<ProductListing />}></Route>
+      <Route path="/wishlist" element={<Wishlist />}></Route>
+      <Route path="/cart-management" element={<Cart />}></Route>
+      <Route path="/login-page" element={<Login />}></Route>
     </Routes>
```
Comment (horsemaker) [inline src/App.js:22]: You could have a separate file for Routes.

File: src/pages/Home/Home.jsx [hunk 4]
```diff
@@ -1,97 +1,123 @@
...
 
-
 const Home = () => {
   return (
     <>
-      <nav className="navigation-container">
-        <div className="nav-brand">
-          <Link className="header-style" to="/">
-            NOZAMA
+      <Nav />
```
Comment (horsemaker) [inline src/pages/Home/Home.jsx]: You could add `Nav` directly in the `App.js` file, outside the Routes. No need to add it to every Screen.

File: src/pages/Home/Home.jsx [hunk 5]
```diff
@@ -1,97 +1,123 @@
...
+            <img
+              className="img-card adjust"
+              src="../assets/home-decor.jpg"
+              alt=""
+            />
+          </div>
         </div>
       </div>
+
+      <Footer />
```
Comment (horsemaker) [inline src/pages/Home/Home.jsx]: You could add `Footer` directly in the `App.js` file, outside the Routes. No need to add it to every Screen.

File: src/pages/Product-listing/product-listing.js [hunk 6]
```diff
@@ -1,9 +1,23 @@
...
+      <Nav />
+
+      <div className="sub-heading title">
+        <h2 className="h2">Shop by Category</h2>
+      </div>
 
+      {/* <h1>Product Listing</h1> */}
 
-}
+      {/* <Footer/> */}
```
Comment (horsemaker) [inline src/pages/Product-listing/product-listing.js]: You could remove this commented code if you are not going to want it later.

**Review by horsemaker — APPROVED** [review]

**Comment by horsemaker** [comment]
Also, try to divide your Home Page into small components to make use of React at its best!

File: src/main.css [hunk 7]
```diff
@@ -3,11 +3,12 @@
 body {
 
     font-family: 'Montserrat', sans-serif;
+    background-color: #fdf2f8;
```
Comment (mimansha-swarup) [inline src/main.css]: can use variable

File: src/pages/Home/Home.jsx (hunk 4 above)
Comment (mimansha-swarup) [inline src/pages/Home/Home.jsx]: place nav before routes in app.js it will be applied over every page

File: src/pages/Home/Home.jsx [hunk 8]
```diff
@@ -1,97 +1,123 @@
...
+            <img
+              className="img-card adjust"
+              src="/assets/pet-supplies.jpg"
+              alt=""
+            />
+          </div>
 
-        <div className="women-card">
-          <p className="card-heading">Women</p>
+          <div className="women-card">
```
Comment (mimansha-swarup) [inline src/pages/Home/Home.jsx:104]: can make component for this card and map it here

**Review by mimansha-swarup — COMMENTED** [review]
also for folders like pages and components, you can have a js file where you can import all other js files of that folder and export it
and use this new js file for importing it will keep importing clean

**Pull request merged**